File: pdlearning/src/microservices/form-microservice/Api/Domain/Entities/QuestionGroup.cs
using System.Diagnostics.CodeAnalysis;
using Thunder.Platform.Core.Domain.Entities;

namespace Microservice.Form.Domain.Entities
{
    [SuppressMessage("Microsoft.Naming", "CA1724", Justification = "Toan Nguyen confirmed this.")]
    public class QuestionGroup : Entity
    {
        public string Name { get; set; }
    }
}

File: pdlearning/src/web-apps/src/libraries/domain-api/src/question-bank/models/question-group.ts
export interface IQuestionGroup {
  id: string;
  name: string;
}

export class QuestionGroup implements IQuestionGroup {
  public id: string;
  public name: string;

  constructor(data?: IQuestionGroup) {
    if (data == null) {
      return;
    }

    this.id = data.id;
    this.name = data.name;
  }
}
